server/routerlicious/packages/services-shared/src/healthCheckEndpoints.ts

```diff
@@ -44,7 +44,6 @@ export function createHealthCheckEndpoints(
 				probeProps,
 			);
 			if (StartupChecker.getInstance().isStartupComplete()) {
-				startupProbeMetric.success("Startup probe successful");
 				response.sendStatus(200);
 			} else {
 				startupProbeMetric.error("Startup probe failed");
```

Comment:
If we're not doing both success and failure, we should just do an `error` log instead of a full metric.

Removing these metrics entirely is probably valid since the HTTP request handler itself is implicitly wrapped in an HTTP Metric that is already going to report Success/Failure and durationInMs based on HTTP response properties. Since these metrics don't have additional telemetry properties added to them, they are not really adding anything on top of the standard HTTP metrics

Reply:
Sg, then I will just log the errors using Lumberjack.